c7n/resources/aws.py

```diff
@@ -229,12 +229,49 @@ class CloudWatchLogOutput(LogOutput):
 
     log_format = '%(asctime)s - %(levelname)s - %(name)s - %(message)s'
 
+    def __init__(self, ctx, config=None):
+        super(CloudWatchLogOutput, self).__init__(ctx, config)
+        if self.config.get('netloc') == 'master':
+            self.log_group = self.config.get('path').strip("/")
+        else:
+            self.log_group = self.config.get('netloc')
+        self.region = self.config.get('region')
+        self.destination = (
+            self.config.scheme == 'aws' and
+            self.config.get('netloc') == 'master') and 'master' or None
+
+    def construct_stream_name(self):
+        if self.config.get('stream') is None:
+            log_stream = self.ctx.policy.name
+            if self.config.get('region') is not None:
+                log_stream = "{}/{}".format(self.ctx.options.region, log_stream)
+            if self.config.get('netloc') == 'master':
+                log_stream = "{}/{}".format(self.ctx.options.account_id, log_stream)
+        else:
+            log_stream = self.config.get('stream').format(
+                region=self.ctx.options.region,
+                account=self.ctx.options.account_id,
```

Comment:
Saw this in the recent release notes, so sorry about commenting on an old PR.  Shouldn't this variable be named `account_id` to match other places that does variable formatting?  Because `account` in other places refers to the account alias name, not the ID.  We should really have consistency across the tool.